lib/api_auth/date_header.ex

```diff
@@ -21,25 +19,65 @@ defmodule ApiAuth.DateHeader do
     hc |> HeaderCompare.compare(@keys, &timestamp_compare/2)
   end
 
+  @doc """
+  Takes a DateTime and returns a string with the date-time in RFC 2616 format.
+  This format is used in the HTTP protocol. Note that the date-time will always be "shifted" to UTC.
+  """
+  def httpdate(dt) do
+    Calendar.strftime(dt, "%a, %d %b %Y %H:%M:%S GMT")
+  end
```

Comment:
confusingly, this `Calendar.strftime` is calling the elixir calendar module, not the calendar package i removed